nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java

```diff
@@ -76,15 +83,12 @@
     @AvroEncode(using = InstantAsLongEncoding.class)
     protected Instant scrollEndTime;
 
-    @Nullable
-    protected Long processed;
+    @AvroEncode(using = InstantAsLongEncoding.class)
```

Comment:
I don't really understand the how the code works but I assume it does!  Here it looks like you are serialising instants as longs with avro?  But elsewhere you have Instants in json, which are serialized as ISO-8601 strings.

Reply:
personally I would have prefer normal timestamp everywhere, but it makes sense to have instant encoded as timestamp with a long instead a string representation that is much more bigger.